common/djangoapps/student/views.py

```diff
@@ -195,6 +195,11 @@ def index(request, extra_context=None, user=AnonymousUser()):
     # 1) Change False to True
     context['show_homepage_promo_video'] = configuration_helpers.get_value('show_homepage_promo_video', False)
 
+    # Maximum number of courses to display on the homepage.
+    context['homepage_course_max'] = configuration_helpers.get_value(
+        'HOMEPAGE_COURSE_MAX', settings.HOMEPAGE_COURSE_MAX
```

Comment:
Is there already a test for the original `settings.HOMEPAGE_COURSE_MAX`?  Do you want a test to ensure that stays the default?

Reply:
Good idea, thanks. I added a couple of extra tests.